cli/tests/integration_tests.rs

```diff
@@ -83,6 +83,9 @@ fn js_unit_tests() {
 mod integration {
   use super::*;
 
+  #[path = "../tools/repl_tests.rs"]
```

Comment:
Should be `./tools/repl_tests.rs` instead?

Reply:
Because it's under the `mod integration` like before then it needs to be `../`. See https://doc.rust-lang.org/reference/items/modules.html#the-path-attribute

Don't know why this doesn't work on the CI... works on linux, mac, and windows for me locally.

Reply:
Behavior change between 1.52 and 1.53 of Rust? That's surprising...

Reply:
No, I wasn't running the command correctly for Linux and Mac. Platform specifc bug. Opened https://github.com/rust-lang/rust/issues/86638